presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestDriver.java

```diff
@@ -73,23 +87,37 @@ public void setup()
         server.createCatalog(TEST_CATALOG, "tpch");
         server.installPlugin(new BlackHolePlugin());
         server.createCatalog("blackhole", "blackhole");
-
+        waitForNodeRefresh(server);
         setupTestTables();
     }
 
+    private static void waitForNodeRefresh(TestingPrestoServer server)
```

Comment:
I took this code from the `StandaloneQueryRunner#refreshNodes()`. `TestingPrestoServer#refreshNodes()` just triggers the refresh but doesn't wait.